GitUI/UserControls/RevisionGrid/FilterInfo.cs

```diff
@@ -339,6 +340,55 @@ public ArgumentString GetRevisionFilter()
             return filter;
         }
 
+        public string FilterSummary()
```

Comment:
```suggestion
        public string GetSummary()
```

Reply:
Not changing in FilterChangedEventArgs.cs (unless requested)

Reply:
In the filter it is a static string, hence the property. Here we run the method to build the summary.